keras_nlp/evaluations/glue/glue.py

```diff
@@ -0,0 +1,287 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import csv
+import os
+
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from absl import app
+from absl import flags
+from tensorflow import keras
+
+import keras_nlp
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string(
+    "task_name",
+    "mrpc",
+    "The name of the GLUE task to finetune on.",
+)
+
+flags.DEFINE_integer(
+    "batch_size",
+    32,
+    "Batch size of data.",
+)
+
+flags.DEFINE_integer(
+    "epochs",
+    2,
+    "Number of epochs to run finetuning.",
+)
+
+flags.DEFINE_float(
+    "learning_rate",
+    5e-5,
+    "Learning rate",
+)
+
+flags.DEFINE_string(
+    "submission_file_path",
```

Comment:
It looks like this a directory from the code? Might want to make that clearer in name/description.

`submission_directory` and `fintuning_model_directory` maybe?